packages/frontend/src/components/project/ContractsSection.tsx

```diff
@@ -48,30 +39,16 @@ export function ContractsSection(props: ContractsSectionProps) {
       <h3 className="font-bold md:text-md">
         The system consists of the following smart contracts:
       </h3>
-      <ul className="list-disc my-4 pl-8 space-y-4">
+      <div className="flex flex-wrap gap-4 my-4">
         {props.contracts.map((contract, i) => (
-          <li key={i}>
-            <strong>{contract.name}</strong>{' '}
-            <EtherscanLink
-              address={contract.address}
-              className="text-xs md:text-base"
+          <React.Fragment key={i}>
```

Comment:
There is no need to wrap this in Fragment just use ContractEntry directly

Reply:
+1